src/site/content/en/blog/codelab-stories-component/index.md

```diff
@@ -1,42 +1,62 @@
 ---
 layout: codelab
-title: Building A Stories Component
+title: Stories component codelab
 authors:
   - adamargyle
 description: |
-  TODO.
-date: 2020-05-21
+  Learn how to build an experience similar to Instagram Stories on the web.
+date: 2020-06-08
 hero: hero.jpg
 glitch: gui-challenges-stories
 glitch_path: app/index.html
 related_post: stories-component
+tags:
+  - css
+  - dom
+  - javascript
+  - layout
+  - mobile
+  - ux
 ---
 
-This is the hands-on experience companion sandbox for this [YouTube video](#). We'll be walking through the HTML, CSS and Javascript, assembling the component as we go! The [associated blog post](#) is great for learning the tradeoffs we made with the strategy.
+This codelab teaches you how to build an experience like Instagram Stories
+on the web. We'll build the component as we go, starting with HTML, then CSS,
+then JavaScript.
 
-{% Aside 'objective' %}
-get our hands dirty with the code to maximize learnings
-{% endAside %}
+Check out my blog post [Building a Stories component on the web](/building-a-stories-component)
+to learn about the tradeoffs I had to make while building this component.
+
+{% YouTube '5_sfnQDr1-o' %}
 
+{% Aside %}
+This post assumes that you're familiar with the Instagram Stories UX
+and will use terminology from that experience (e.g. a "friend's story").
+{% endAside %}
 
-### Introduction
-We'll be first going over the HTML, then the CSS, and last the Javascript. By the end you'll have a working component that you can use the same strategy for on something else.
+## Setup
 
-Prep
-- open `app/index.html`
+{% Instruction 'remix', 'ol' %}
+1. Open `app/index.html`.
 
 ## HTML
-Since each friend can have any number of stories, I thought it was meaningful to make my friends `<section>`'s and each of their stories `<article>`'s. Let's start from the beginning though. 
 
-We need a container to start from, this will be our stories component. Our first lines of code are a simple div with a class of stories. Paste the follow code inside of our `<body>` tag of `app/index.html`.
+I always aim to use [semantic HTML](https://en.wikipedia.org/wiki/Semantic_HTML).
+Since each friend can have any number of stories, I thought it was meaningful to use a
+`<section>` element for each friend and an `<article>` element for each story.
+Let's start from the beginning though. First, we need a container for our
+stories component.
+
+Add a `<div>` element to your `<body>`:
 
-```html/0-2
+```html/0,2
```

Comment:
i searched the handbook for this syntax, is it there!? cuz it's rad, ty

Reply:
Looks like we're missing it. I created #3229 to document it.